[Bug]: Cannot find module @deepgram/cli/src/hooks/init/log

### Contact Details

_No response_

### What happened?

Happens right after install.
```
user@hp-laptop:~$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
user@hp-laptop:~$ node --version
v18.18.0
user@hp-laptop:~$ npm --version
9.8.1
user@hp-laptop:~$ deepgram
 ›   ModuleLoadError: [MODULE_NOT_FOUND] require failed to load /home/user/.nvm
 ›   /versions/node/v18.18.0/lib/node_modules/@deepgram/cli/src/hooks/init/log:
 ›    Cannot find module '/home/user/.nvm/versions/node/v18.18.0/lib/node_modul
 ›   es/@deepgram/cli/src/hooks/init/log'
 ›   Require stack:
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/module-loader.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/help/util.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/help/formatter.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/help/command.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/help/index.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/flags.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/cli-ux/styled/table.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/cli-ux/styled/index.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/cli-ux/index.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/command.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/no
 ›   de_modules/@oclif/core/lib/index.js
 ›   - /home/user/.nvm/versions/node/v18.18.0/lib/node_modules/@deepgram/cli/bi
 ›   n/run
 ›   Code: MODULE_NOT_FOUND

```

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/package.json
+++ b/package.json
@@ -26,11 +26,7 @@
       "@oclif/plugin-help",
       "@oclif/plugin-not-found"
     ],
-    "topicSeparator": " ",
-    "hooks": {
-      "init": "./src/hooks/init/log",
-      "command_not_found": "./src/hooks/command_not_found/log"
-    }
+    "topicSeparator": " "
   },
   "scripts": {
     "build": "shx rm -rf dist && tsc -b",

--- a/src/hooks/command_not_found/log.ts
+++ b/src/hooks/command_not_found/log.ts
@@ -1,8 +1,0 @@
-import { Hook } from "@oclif/core";
-
-const hook: Hook<"command_not_found"> = async function (opts) {
-  process.stdout.write(`command "${opts.id}" not found\n`);
-  // this.exit();
-};
-
-export default hook;

--- a/src/hooks/init/log.ts
+++ b/src/hooks/init/log.ts
@@ -1,7 +1,0 @@
-import { Hook } from "@oclif/core";
-
-const hook: Hook<"init"> = async function (opts) {
-  // process.stdout.write(`example hook running ${opts.id}\n`)
-};
-
-export default hook;

--- a/test/hooks/command_not_found/log.test.ts
+++ b/test/hooks/command_not_found/log.test.ts
@@ -1,9 +1,0 @@
-import {expect, test} from '@oclif/test'
-
-describe('hooks', () => {
-  test
-  .stdout()
-  .hook('init', {id: 'mycommand'})
-  .do(output => expect(output.stdout).to.contain('example hook running mycommand'))
-  .it('shows a message')
-})

--- a/test/hooks/init/log.test.ts
+++ b/test/hooks/init/log.test.ts
@@ -1,9 +1,0 @@
-import {expect, test} from '@oclif/test'
-
-describe('hooks', () => {
-  test
-  .stdout()
-  .hook('init', {id: 'mycommand'})
-  .do(output => expect(output.stdout).to.contain('example hook running mycommand'))
-  .it('shows a message')
-})

--- a/test/hooks/postrun/log.test.ts
+++ b/test/hooks/postrun/log.test.ts
@@ -1,9 +1,0 @@
-import {expect, test} from '@oclif/test'
-
-describe('hooks', () => {
-  test
-  .stdout()
-  .hook('init', {id: 'mycommand'})
-  .do(output => expect(output.stdout).to.contain('example hook running mycommand'))
-  .it('shows a message')
-})

--- a/test/hooks/prerun/auth.test.ts
+++ b/test/hooks/prerun/auth.test.ts
@@ -1,9 +1,0 @@
-import {expect, test} from '@oclif/test'
-
-describe('hooks', () => {
-  test
-  .stdout()
-  .hook('init', {id: 'mycommand'})
-  .do(output => expect(output.stdout).to.contain('example hook running mycommand'))
-  .it('shows a message')
-})

